Networking/Poller.cc

```diff
@@ -16,26 +16,37 @@
 // limitations under the License.
 //
 
+#define NOMINMAX
 #include "Poller.hh"
 #include "Error.hh"
 #include "Logging.hh"
 #include "ThreadUtil.hh"
 #include "PlatformIO.hh"
 #include "c4Base.h"
+#include "sockpp/platform.h"
+#include "sockpp/tcp_acceptor.h"
+#include "sockpp/tcp_connector.h"
 #include <vector>
 #include <errno.h>
-#include <poll.h>
 
 #ifndef _WIN32
 #include <unistd.h>
+#include <poll.h>
+#define tcp_read ::read
+#define tcp_write ::write
+#define tcp_poll ::poll(fdArray, nfds_t(fdSize), timeout)
+#else
+#define tcp_read(s, buf, len) ::recv(s, (char *)(buf), len, 0)
```

Comment:
Since`tcp_read` and `tcp_write` are only called once apiece, I think it'd be cleaner to just use an `#ifdef` at the point of the call, rather than defining these macros. That way it matches the ifdef where we create the interrupting FDs.